projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/components/google-drive-connect-button.tsx

```diff
@@ -0,0 +1,50 @@
+import requestExternalAccess from '@automattic/request-external-access';
+import { Button } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+import { __, _x } from '@wordpress/i18n';
+
+type Props = {
+	settingsUrl?: string;
+	onConnected: () => void;
+	isConnected: boolean;
+};
+
+export default function GoogleDriveConnectButton( {
+	settingsUrl,
+	onConnected,
+	isConnected,
+}: Props ) {
+	const [ isToggling, setIsToggling ] = useState( false );
+
+	useEffect( () => {
+		// Reset local UI state when connection flips
+		setIsToggling( false );
+	}, [ isConnected ] );
+
+	const handleClick = () => {
+		if ( ! settingsUrl ) return;
+		setIsToggling( true );
+		requestExternalAccess( settingsUrl, ( { keyring_id: keyringId } ) => {
+			if ( keyringId ) {
+				onConnected();
+			} else {
+				setIsToggling( false );
+			}
+		} );
+	};
+
+	return (
+		<Button
+			variant="secondary"
+			onClick={ handleClick }
+			target="_blank"
+			rel="noopener noreferrer"
+			__next40pxDefaultSize={ true }
+			disabled={ ! settingsUrl || isToggling }
+		>
+			{ isToggling
+				? __( 'Connecting…', 'jetpack-forms' )
+				: _x( 'Connect to Google Drive', '', 'jetpack-forms' ) }
```

Comment:
There's `_x()` but context is empty; seems fine to just use `__()`

```suggestion
				: __( 'Connect to Google Drive', 'jetpack-forms' ) }
```

Reply:
Fixed.